### PR TITLE
[onert] Refactor ir::Graph::sweepGarbageOperands

### PR DESCRIPTION
**Pull request by wateret**
Make "sweeping garbage operands" as a graph pass.

- Remove method `ir::Graph::sweepGarbageOperands`
- Introudce a pass `compiler::pass::UnusedOperandEliminationPass`

ONE-DCO-1.0-Signed-off-by: Hanjoung Lee <hanjoung.lee@samsung.com>

### CODE REVIEW

File: runtime/onert/test/graph/operand/LayoutSet.cc [hunk 1]
```diff
@@ -36,6 +36,15 @@ TEST(ir_LayoutSet, neg_add_remove)
...
 }
 
+TEST(ir_LayoutSet, neg_add_twice)
+{
+  LayoutSet set;
+  set.add(Layout::NHWC);
+  ASSERT_EQ(set.size(), 1);
+  set.add(Layout::NHWC);
+  ASSERT_EQ(set.size(), 1);
+}
```
Comment (wateret) [inline runtime/onert/test/graph/operand/LayoutSet.cc:46]: An unrelated negative test case is added to meet our test case addition rules.

**Review by seanshpark — DISMISSED** [review]
LGTM

**Comment by wateret** [comment]
Updated - removed a unused line (trivial)

**Review by seanshpark — APPROVED** [review]
LGTM

**Pull request merged**